### PR TITLE
Update jtreg to 5.1 for jdk 15 and 15+

### PR DESCRIPTION
**Pull request by sophia-guo**
Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

File: openjdk/build.xml [hunk 1]
```diff
@@ -27,38 +27,50 @@
 
 	<target name="getJtreg">
 		<mkdir dir="${DEST}"/>
+		<if>
+			<or>
+				<contains string="${JDK_VERSION}" substring="8"/>
```
Comment (karianna) [inline openjdk/build.xml:32]: Ignoring 9-10?

**Comment by sophia-guo** [comment]
Updated to include all versions.

File: openjdk/build.xml [hunk 2]
```diff
@@ -27,38 +27,50 @@
 
 	<target name="getJtreg">
 		<mkdir dir="${DEST}"/>
+		<if>
+			<or>
+				<matches pattern="^[89]{1}$" string="${JDK_VERSION}"/>
+				<matches pattern="^[1][01234]" string="${JDK_VERSION}"/>
```
Comment (smlambert) [inline openjdk/build.xml:33]: Look at you with your pattern matching!

**Review by smlambert — APPROVED** [review]
thanks @sophia-guo !

**Pull request merged**